packages/gamut-labs/src/experimental/Box/types.ts

```diff
@@ -0,0 +1,5 @@
+import { spacing } from '@codecademy/gamut-styles';
+
+export type BoxVariants = 'yellow' | 'navy' | 'white';
```

Comment:
as mentioned above, this could instead be:
```suggestion
import { BOX_VARIANTS } from './constants';

export type BoxVariants = keyof typeof BOX_VARIANTS;
```